src/lib/modules/console/index.ts

```diff
@@ -96,10 +101,18 @@ class Console {
         "    var all/v a/va - " + __("During a debug, display all variables"),
         "log <process> on/off - " + __("Activate or deactivate the logs of a sub-process. Options: blockchain, "),
         "plugin install <package> - " + __("Installs a plugin in the Dapp. eg: plugin install embark-solc"),
-        "quit - " + __("to immediatly exit (alias: exit)"),
-        "",
-        __("The web3 object and the interfaces for the deployed contracts and their methods are also available"),
       ];
+      helpDescriptions.forEach((helpDescription) => {
+        let matches = [] as string[];
+        if (typeof helpDescription.matches === "object") {
+          matches = helpDescription.matches as string[];
```

Comment:
This throws me off a bit... so if `helpDescription.matches` is an object, we want to assign it as `string[]` ?

Can you elaborate what this is about? According to `HelpDescription` matches is either `string[]` or `MatchFunction`..

Reply:
In JS, an array is a type `object` 🤷‍♂️

Reply:
Ah, so if you wanna check for array then better use `Array.isArray(arr)`

Reply:
True, forgot about that one.